service/history/workflowTaskHandler.go

```diff
@@ -813,10 +813,16 @@ func (handler *workflowTaskHandlerImpl) handleCommandContinueAsNewWorkflow(
 		return err
 	}
 
-	if err := searchattribute.SubstituteAliases(handler.searchAttributesMapper, attr.GetSearchAttributes(), namespaceName.String()); err != nil {
+	mappedSearchAttributes, err := searchattribute.SubstituteAliases(handler.searchAttributesMapper, attr.GetSearchAttributes(), namespaceName.String())
+	if err != nil {
 		handler.stopProcessing = true
 		return err
 	}
+	if mappedSearchAttributes != nil {
+		newAttr := *attr
```

Comment:
:scream:  I thought "attr" wasn't allowed

Reply:
:-) This is another `attr`. This is command attributes, not search.